igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy

```diff
@@ -38,14 +39,7 @@ class DockerRegistryCache {
         Jedis resource = jedisPool.resource
         def key = "${prefix}:${id}:${account}*"
 
-        def res = resource.keys(key).collect {
-            def parse = parseKey(it)
-            if (parse) {
-                formatImage(parse.registry, parse.repository, parse.tag)
-            } else {
-                null
-            }
-        }.findAll { it != null }
+        List<String> res = resource.keys(key).findAll { it != null } as List
```

Comment:
Could also use:

`res = resource.keys(key).findResults( { it } )`

or

`res = resource.keys(key) - null`